module/importer/src/Infrastructure/Handler/DeleteSourceCommandHandler.php

```diff
@@ -30,6 +46,18 @@ public function __invoke(DeleteSourceCommand $command): void
 
         Assert::notNull($source, sprintf('Can\'t fid source "%s"', $command->getId()->getValue()));
 
+        $fileNames = $this->importQuery->getFileNamesBySourceId($source->getId());
+
         $this->sourceRepository->delete($source);
+
+        foreach ($fileNames as $fileName) {
```

Comment:
Sam problem like on exports probably problem when source will work differently and no deletion of import and errors